konduit-serving-cli/src/test/java/ai/konduit/serving/cli/data/JsonCoverageTest.java

```diff
@@ -0,0 +1,28 @@
+package ai.konduit.serving.cli.data;
+
+import ai.konduit.serving.common.test.BaseJsonCoverageTest;
+import ai.konduit.serving.pipeline.util.ObjectMappers;
+
+public class JsonCoverageTest extends BaseJsonCoverageTest {
+
+
+    @Override
+    public String getPackageName() {
+        return "ai.konduit.serving.cli";
+    }
+
+    @Override
+    public Object fromJson(Class<?> c, String json) {
+        return ObjectMappers.fromJson(json, c);
+    }
+
+    @Override
+    public Object fromYaml(Class<?> c, String yaml) {
+        return ObjectMappers.fromYaml(yaml, c);
+    }
+
+
+
```

Comment:
For all modules (currently) without anything that extends TextConfig (i.e., any JsonCoverageTest without an `@Test`) I think we'll need at least one test to be implemented in the class for the `@AfterClass` to run.
Something like this should suffice:
```
@Test
public void test(){
    //Empty test to ensure @Before is run at least one and @AfterClass is run
}
```